README.md

```diff
@@ -19,72 +19,41 @@ against the upstream Elasticsearch instance using Kibana's credentials**.
 This is probably what you want if you're deploying Kibana, but it means you
```

Comment:
README one paragraph up (out of the diff, so I can't comment there) says:

>  Kibana as an Aptible app. This app automatically detects your Elasticsearch
> version and starts Kibana 4.1, 4.4, or 5.0 accordingly.

Can you update accordingly? (the latter is no longer correct)